docs/source/daml/intro/daml/3_Data/List.daml

```diff
@@ -0,0 +1,28 @@
+daml 1.2
+module List where
+
+-- LIST_TEST_BEGIN
+import DA.List
+
+list_test = scenario do
+  let
+    empty : [Int] = []
```

Comment:
The textual documentation uses `List Int` rather than `[Int]`; this should probably be mentioned as an alternative syntax.

Reply:
`List Int` is deprecated. I'm changing the text.